packages/package.json

```diff
@@ -42,7 +42,7 @@
     "react-responsive": "^9.0.2",
     "react-slider": "^2.0.6",
     "react-table": "^7.8.0",
-    "react-virtualized-auto-sizer": "^1.0.20",
+    "react-virtualized-auto-sizer": "1.0.20",
```

Comment:
Had to remove the `^` because a patch release broke the tests 😕 I raised it with the owners https://github.com/bvaughn/react-virtualized-auto-sizer/issues/80